philadelphia/src/main/java/org/jvirtanen/philadelphia/FIXStatusListener.java

```diff
@@ -10,57 +10,64 @@
     /**
      * Receive an indication to close the connection.
      *
+     * @param session the Session
```

Comment:
A small nit, but for uniformity, could you use lowercase for "session" (cf. "message" below)? Thanks for bearing with me. :grin: